src/app/(Authentication)/Login/Email/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { router } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import AuthInput from '../../../../Components/AuthInput/AuthInput';
+import { emailExists } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [email, setEmail] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const [displayEmail, setDisplayEmail] = useState(false);
```

Comment:
Add a `placeholder` `useState`, add this to the `AuthInputProps` in the `AuthInput` component, and pass down the variable and its function.